packages/utils/types.ts

```diff
@@ -21,4 +21,4 @@ export type PartialReturnType<T extends (...args: unknown[]) => unknown> = Parti
 
 export type SFCWithInstall<T> = T & Plugin
 
-export type RefElement = Nullable<HTMLElement>
+export type RefElement = HTMLElement | undefined
```

Comment:
Can we put a `null` here after `undefined` might be more accurate.

Reply:
Sure, I changed it.